src/vanilla.ts

```diff
@@ -357,7 +357,7 @@ export function getVersion(proxyObject: unknown): number | undefined {
 
 export function subscribe<T extends object>(
   proxyObject: T,
-  callback: (ops: Op[]) => void,
+  callback: (ops: Op[]) => void | Promise<void>,
```

Comment:
I don't think it's necessary. If it does, something seems wrong.
```suggestion
  callback: (ops: Op[]) => void,
```

Reply:
You're right the promise return value isn't being used anyways